lib/versioned/^4.0.0/index.js

```diff
@@ -33,16 +34,18 @@ function execute(opts, env, config) {
   logSyncTask(gulpInst, opts);
 
   // This is what actually loads up the gulpfile
-  var exported = require(env.configPath);
+  requireOrImport(env.configPath, function(err, exported) {
+    if (err) {
+      console.error(err);
```

Comment:
Previously, if `require()` failed we got an unhandled exception on the module level. So I use `console.error()`+`exit()` here to mimick the old behavior as close as possible, or would you rather log this error differently? (I noticed that you generally seem to use `gulplog`).

Also I'm aware that I still have to do similar changes to the other `lib/versioned/*/index.js` scripts. But I'd first like to get your feedback on the general approach before moving forward. What do you think?

Reply:
I think this works well! Good solution :)